test/cases/api.sh

```diff
@@ -799,6 +815,22 @@ function verifyInAWS() {
   # Verify image
   _ssh="ssh -oStrictHostKeyChecking=no -i ./keypair.pem $SSH_USER@$HOST"
   _instanceCheck "$_ssh"
+
+  # Check access to user1 and user2
+  check_groups=$(ssh -i /tmp/usertest "user1@$HOST" -t 'groups')
```

Comment:
Could also check straight up with `ssh user1@HOST sudo echo I have sudo`, but either is fine I suppose

Reply:
I tried to, but sudo asks for user1 password (which I don't know also).